index.html

```diff
@@ -9,6 +9,16 @@
  <body>
     <h1> Amber Hinojos </h1>
 
+    <nav class="navbar"> 
+        <ul>
+            <li><a href=""> About </a></li>
+            <li><a href=""> Experience </a></li>
+            <li><a href=""> Connect </a></li>
+            <li><a href=""> Skills </a></li>
+            <li><a href=""> Projects </a></li>
```

Comment:
The href's are empty.  They should reference the id on each section.  e.g. href='#Experience".